cvat-ui/src/components/bulk-progress.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Progress, Button } from 'antd';
```

Comment:
Maybe this is not necessary anymore (need to double check), but let's stick to current style of importing each component from `antd` separately.